app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/MainActivity.java

```diff
@@ -376,7 +372,6 @@ private void showChangelog() {
 
     @Override
     protected void reload() {
-        // TODO Propagate reload to fragments?
```

Comment:
Fixed by implementing reload of different tab pages + call it from the base class of all tab pages